modules/ROOT/pages/release-notes/release-notes5014.adoc

```diff
@@ -14,7 +14,7 @@
 
 The autocompleter API now provides a new `reload` function to allow the autocompleter menu to reload, with potentially more or different results. This could be used for a "More..." menu item that would load more results.
 
-See the autocompleter documentation for information: https://www.tiny.cloud/docs/ui-components/autocompleter/[\https://www.tiny.cloud/docs/ui-components/autocompleter/]
+See the autocompleter documentation for information: https://www.tiny.cloud/docs/ui-components/autocompleter/[https://www.tiny.cloud/docs/ui-components/autocompleter/]
```

Comment:
This should be an xref instead and the URL text will need updating.
```suggestion
See the autocompleter documentation for information: xref:ui-components/autocompleter.adoc[UI components - Autocompleter]
```

Reply:
done